### DISCUSSION TITLE
How show first tab

### DISCUSSION THREAD

**Discussion by rafleboss47** [roubachof/Sharpnado.Tabs#88] [General]
Hello,

I've just implemented tabs in my project.
How do I pre-select the first TAB?
By default, no tabs are selected.

I use the following system:

`<tabs:TabHostView x:Name="TabHost"
                      Grid.Row="1"
                      Margin="0"
                      BackgroundColor="Black"
                      SelectedIndex="{Binding Source={x:Reference Switcher}, Path=SelectedIndex, Mode=TwoWay}"
                      TabType="Fixed"
                      >
        <tabs:TabHostView.Tabs >
            <tabs:UnderlinedTabItem IsSelected="True" IsSelectable="True" x:Name="onglet1" Label="MMM"  SelectedTabColor="White" UnselectedLabelColor="Gray"/>
            <tabs:UnderlinedTabItem Label="YYYY" SelectedTabColor="White" UnselectedLabelColor="Gray"/>
        </tabs:TabHostView.Tabs>
    </tabs:TabHostView>

    <ScrollView Grid.Row="2">
        <tabs:ViewSwitcher x:Name="Switcher"
                           Animate="True"                           
                           SelectedIndex="{Binding SelectedViewModelIndex, Mode=TwoWay}"
                           TabIndex="1"
                           >
                   <StackLayout  Margin="0" Padding="0" BackgroundColor="White" Spacing="0">
                    <Label TextColor="Black" Margin="0"  Padding="10,0,10,0" Text="RENDEZ-VOUS PASSES" FontSize="Large" VerticalOptions="CenterAndExpand" HorizontalOptions="Center"></Label>
                    <StackLayout  HeightRequest="6" BackgroundColor="Black"  Margin="0"></StackLayout>
                </StackLayout>

<StackLayout  Margin="0" Padding="0" BackgroundColor="White" Spacing="0">
                    <Label TextColor="Black" Margin="0"  Padding="10,0,10,0" Text="RENDEZ-VOUS PASSES" FontSize="Large" VerticalOptions="CenterAndExpand" HorizontalOptions="Center"></Label>
                    <StackLayout  HeightRequest="6" BackgroundColor="Black"  Margin="0"></StackLayout>
                </StackLayout>

</tabs:Shadows>
`

**Comment by roubachof** [comment]
Set the SelectedViewModelIndex to the index you want
